doctrine/pdo_session_storage.rst

```diff
@@ -63,6 +63,11 @@ To use it, first register a new handler service:
                 array('db_username' => 'myuser', 'db_password' => 'mypassword')
             ))
         ;
+.. caution::
```

Comment:
I like this caution a lot because it's something important and can cause lots of problems. However, I'd like it to be more actionable. It barely explains the problem but it doesn't explain the solution.

My main concerns are:

* Which "lock modes" are we talking about here? Database, Symfony, Doctrine? It it's Doctrine, we can add a link to https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/transactions-and-concurrency.html#locking-support
* *"If you use the default value..."*  can we please explain which is the default value?
* *"... it causes an exception"*  OK, but what can I do to fix that? Any of the other Doctrine lock values solve the problem?

Thanks!

Reply:
@Teshynil @ricknox what could we do about my previous comment? Thanks!